packages/@aws-cdk/aws-stepfunctions-tasks/lib/emr/emr-create-cluster.ts

```diff
@@ -191,6 +201,18 @@ export class EmrCreateCluster extends sfn.TaskStateBase {
     }
 
     this.taskPolicies = this.createPolicyStatements(this._serviceRole, this._clusterRole, this._autoScalingRole);
+
+    if (this.props.stepConcurrencyLevel) {
```

Comment:
add tests for this validation

Reply:
The validation fails when `stepConcurrencyLevel = 0`. Updated to `this.props.stepConcurrencyLevel !== undefined`. Good catch :).